Controller/Payment/FormDataAnalytics.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace Razorpay\Magento\Controller\Payment;
+
+use Razorpay\Magento\Model\Config;
+use Magento\Framework\Controller\ResultFactory;
+use Razorpay\Magento\Model\TrackPluginInstrumentation;
+use Razorpay\Magento\Model\PaymentMethod;
+
+
+/**
+ * Using TrackPluginInstrumentation and sending events to Segment
+ *
+ * ...
+ */
+class FormDataAnalytics extends \Razorpay\Magento\Controller\BaseController
+{
+    protected $setup;
+
+    protected PaymentMethod $paymentMethod;
```

Comment:
@nikhilsharma1100 php error thrown at this line, please check its use and execution.

Reply:
removed this class object as it is not being used